Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  OnboardingViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol OnboardingViewControllerDelegate {
+    func onboardingViewControllerDidTapNextButton(viewController: OnboardingViewController)
+}
+
+class OnboardingViewController: UIViewController {
+
+    internal var stackView: UIStackView!
+
+    internal var titleLabel: UILabel!
+    internal var subtitleLabel: UILabel!
+    internal var imageView: UIImageView!
+    internal var nextButton: UIButton!
+
+    var delegate: OnboardingViewControllerDelegate?
+
+    let model: OnboardingModel!
+
+    init(model: OnboardingModel,
+         nibName nibNameOrNil: String?,
+         bundle nibBundleOrNil: Bundle?) {
+      self.model = model
+      super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+      fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.view.backgroundColor = .eateryBlue
+
+        stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.distribution = .fill
+        stackView.alignment = .center
+        stackView.spacing = 40
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        self.view.addSubview(stackView)
+        stackView.snp.makeConstraints { make in
+            make.center.equalToSuperview()
+            make.width.equalToSuperview().inset(30)
+        }
+
+        setUpTitleLabel()
+        setUpSubtitleLabel()
+        setUpImageView()
+        setUpButton()
+    }
+
+    private func setUpTitleLabel() {
+        titleLabel = UILabel(frame: CGRect.zero)
+        titleLabel.text = model.title
+        titleLabel.textColor = .white
+        titleLabel.textAlignment = .center
+        titleLabel.font = UIFont.systemFont(ofSize: 36, weight: .bold)
```

Comment:
can we add this to an extension of `UIFont` as:
```
extension UIFont {
  static let _36BoldFont = UIFont.systemFont(ofSize: 36, weight: .bold)
}
```
We might already have this extension for `Eatery`

Reply:
We don't have an extension file for fonts?